cmd/mdmctl/apply_dep_profile.go

```diff
@@ -16,15 +18,23 @@ func (cmd *applyCommand) applyDEPProfile(args []string) error {
 	var (
 		flProfilePath = flagset.String("f", "", "filename of DEP profile to apply")
 		flTemplate    = flagset.Bool("template", false, "print a JSON example of a DEP profile")
+		flAnchorFile  = flagset.String("anchor", "", "filename of PEM certificate to add to trusted anchors")
```

Comment:
Hmm, what do you mean a valid chain? In the case of a self-signed cert a single cert is the "valid chain." Happy to modify the text though. What do you suggest?